libmdnsd/CMakeLists.txt

```
@@ -46,8 +46,12 @@ endif()
 configure_file("mdnsd_config.h.in" "${PROJECT_BINARY_DIR}/mdnsd_config.h")
 include_directories(${PROJECT_BINARY_DIR})
 
-set(LIBRARY_FILES mdnsd.c mdnsd.h 1035.c 1035.h sdtxt.c sdtxt.h xht.c xht.h)
+set(PUBLIC_INCLUDES mdnsd.h 1035.h sdtxt.h xht.h)
+if(WIN32)
```

I suppose the `ms_stdint.h` header is for the Visual Studio compiler only?
Then
```suggestion
if(CMAKE_C_COMPILER_ID MATCHES MSVC)
```
See https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html